README.md

```diff
@@ -1,127 +1,177 @@
+| `:FindIdentifierDeclaration` arg   | Open to the identifier `arg` declaration.                  | [Identifier information](#identifier-information)        | ocaml.nvim |
+| `:DocumentIdentifier` arg          | Display the identifier `arg` documentation.                | [Identifier information](#identifier-information)        | ocaml.nvim |
+
+### Details
+
+#### Holes handling
+Typed holes (`_`) are incomplete parts of your OCaml code where the compiler expects an expression.
+- `:JumpPrevHole` jumps to the previous hole in the buffer.
+- `:JumpNextHole` jumps to the next hole in the buffer.
+- `:Construct` opens a list of valid expressions that can replace the hole under the cursor. Type inference is required to propose correct substitutions.
 
 ![Construct example](media/construct.gif)
 
-### Source browsing
-
-Allows you to navigate semantically in a buffer, passing from an expression to
-the parent `let`, the parent `module`, the parent `fun` and the parent `match` expression.
-It is also possible to navigate between pattern matching cases:
-
-- `Jump`: jumps to the referenced expression
+#### Code navigation
+These functions help you navigate semantically in the buffer.
+- `:Jump arg` jumps to the expression or declaration referenced by `arg`. For example, this could be `:Jump let` to jump to the
 
 ![Jump example](media/jump.gif)
 
-- `PhrasePrev`: jump to the beginning of the previous phrase
-- `PhraseNext`: jump to the beginning of the next phrase
+A phrase in OCaml is a complete syntactic unit, such as a definition, expression, or declaration, that can be evaluated or compiled independently.
+- `:PhrasePrev` jumps to the beginning of the previous phrase in the code.
+- `:PhraseNext` jumps to the beginning of the next phrase.
 
 ![Phrase example](media/phrase.gif)
 
-### Infer interface
-
-Used to infer the type of an interface file. If the buffer is not empty,
-a prompt will ask for confirmation to overwrite the buffer contents.
+#### Interface file management
+These functions help alternate between the `.ml` and `.mli` files and infer the interface.
+- `:InferIntf` infer the type of an interface file from its implementation.
 
 ![Infer example](media/infer.gif)
 
-### Find alternate file
-
-Quickly switch from the implementation file to the interface file and
-vice versa. If the interface file does not exist, a prompt can be used to
-generate it.
-
-![Alternate file example](media/switch.gif)
+- `:AlternateFile` switch from the implementation file to the interface file and vice versa.
 
-### Find identifiers definition/declaration
+#### Identifier information
+These functions provide information about a specific identifier specified by `arg`.
+`arg` could be, for example, `List.hd` or `Sys.time`.
+- `:FindIdentifierDefinition arg` opens the identifier `arg` definition.
```

Comment:
Not sure I understand what is happening there.
Isn't it that it opens the file containing the definition of `arg` in a split window?